### PR TITLE
add other middleware config

### PR DESCRIPTION
**Pull request by fu-turer**

### CODE REVIEW

**Comment by fu-turer** [comment]
image

File: src/main/java/com/github/shoothzj/dev/module/config/CassandraConfig.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+@Setter
+@Getter
+public class CassandraConfig extends BaseConfig {
+
+    private String k8sName;
+
+    private String namespace;
+
+    private String deployName;
```
Comment (goflutterjava) [inline src/main/java/com/github/shoothzj/dev/module/config/CassandraConfig.java:33]: statefulset

File: src/main/java/com/github/shoothzj/dev/storage/StorageBookkeeper.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+    private static final StorageBookkeeper INSTANCE = new StorageBookkeeper();
+
+    public static StorageBookkeeper getInstance() {
+        return INSTANCE;
+    }
+
+    @Override
+    protected String getConfigPath() {
+        return StorageUtil.PULSAR_CONF_PATH;
```
Comment (goflutterjava) [inline src/main/java/com/github/shoothzj/dev/storage/StorageBookkeeper.java:38]: path

File: src/main/java/com/github/shoothzj/dev/storage/StorageLvs.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+    private static final StorageLvs INSTANCE = new StorageLvs();
+
+    public static StorageLvs getInstance() {
+        return INSTANCE;
+    }
+
+    @Override
+    protected String getConfigPath() {
+        return StorageUtil.PULSAR_CONF_PATH;
```
Comment (goflutterjava) [inline src/main/java/com/github/shoothzj/dev/storage/StorageLvs.java:38]: path

File: src/main/java/com/github/shoothzj/dev/storage/StorageMysql.java [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+    private static final StorageMysql INSTANCE = new StorageMysql();
+
+    public static StorageMysql getInstance() {
+        return INSTANCE;
+    }
+
+    @Override
+    protected String getConfigPath() {
+        return StorageUtil.PULSAR_CONF_PATH;
```
Comment (goflutterjava) [inline src/main/java/com/github/shoothzj/dev/storage/StorageMysql.java:38]: path

File: src/main/java/com/github/shoothzj/dev/storage/StorageRedis.java [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+    private static final StorageRedis INSTANCE = new StorageRedis();
+
+    public static StorageRedis getInstance() {
+        return INSTANCE;
+    }
+
+    @Override
+    protected String getConfigPath() {
+        return StorageUtil.PULSAR_CONF_PATH;
```
Comment (goflutterjava) [inline src/main/java/com/github/shoothzj/dev/storage/StorageRedis.java:38]: path

File: src/main/java/com/github/shoothzj/dev/storage/StorageUtil.java [hunk 6]
```diff
@@ -39,12 +39,30 @@
 
     public static final String ZK_CONF_PATH = STORAGE_PREFIX + "zookeeper-conf.json";
 
+    public static final String PULSAR_CONF_PATH = STORAGE_PREFIX + "pulsar-conf.json";
+
+    public static final String BOOKKEEPER_CONF_PATH = STORAGE_PREFIX + "bookkeeper-conf.json";
```
Comment (goflutterjava) [inline src/main/java/com/github/shoothzj/dev/storage/StorageUtil.java:44]: please sort by alphabeta

File: src/main/kotlin/widget/config/ConfigBookkeeper.kt [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+    val editK8sInstanceName = mutableStateOf("")
+    val editBookkeeperNamespace = mutableStateOf("default")
+    val editBookkeeperStatefulsetName = mutableStateOf("bookkeeper")
+    val kubernetes = StorageK8s.getInstance().listConfigNames()
+    ConfigBase(
+        "bookkeeper",
+        dialogState,
+        errorState,
+        dialogInputContent = {
+            ConfigItemString(editBookkeeperName, R.strings.name)
```
Comment (goflutterjava) [inline src/main/kotlin/widget/config/ConfigBookkeeper.kt:45]: make `singleLine` to true to leverage `tab` key

**Comment by fu-turer** [comment]
image

**Pull request merged**